src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -473,6 +481,16 @@ public void setAllowEmptyCatches(boolean allow) {
         allowEmptyCatches = allow;
     }
 
+    /**
+     * Setter to allow empty switch case and default bodies.
```

Comment:
```suggestion
     * Setter to allow empty switch case and default blocks and block statements.
```

Reply:
I started with this wording but then I decided to change it to keep consistency with the other properties (all have the word "bodies"). What do you think?

```
    /** Allow empty constructor bodies. */
    private boolean allowEmptyConstructors;
    /** Allow empty method bodies. */
    private boolean allowEmptyMethods;
    /** Allow empty class, interface and enum bodies. */
    private boolean allowEmptyTypes;
    /** Allow empty loop bodies. */
    private boolean allowEmptyLoops;
    /** Allow empty lambda bodies. */
    private boolean allowEmptyLambdas;
    /** Allow empty catch bodies. */
    private boolean allowEmptyCatches;
    /** Allow empty switch case and default bodies. */
    private boolean allowEmptySwitchCase;
```

Reply:
It's not that big a deal, but I would prefer being "correct" than consistent in this case. Up to you.